elastic/kibana
# [Cloud Security] telemetry for agentless

solves:
- https://github.com/elastic/security-team/issues/8253
## Summary

Collect telemetry when customer is using agentless policy.



## Demo
In the image bellow, you can see that CSPM is installed on two different agent policy, the first once is the regular one and the second one is the agentless policy.
<img width="1342" alt="image" src="https://github.com/elastic/kibana/assets/90558359/2a0b438b-7a8a-4f7c-a47f-8d864e45a211">

In the `installation_stats` telemetry payload, we can now see a new field called 'is_agentless' which indicates whether the installation is agentless or not.

<img width="500" alt="image" src="https://github.com/elastic/kibana/assets/90558359/38c20e17-8197-44ec-b2bf-2208db2c09d0">

## How to test
Install CSPM on agentless policy and on non-agentless policy and call the [telemetry api](https://docs.elastic.dev/security-solution/cloud-security/telemetry/telemetry_pipeline_overview#check-telemetry-output-in-kibana) via the dev tools.
